modules/crafted-completion-config.el

```diff
@@ -43,22 +41,32 @@ ARG is the thing being completed in the minibuffer."
     (with-eval-after-load 'evil
       (keymap-set vertico-map "C-j" 'vertico-next)
       (keymap-set vertico-map "C-k" 'vertico-previous)
-      (keymap-set vertico-map "M-h" 'vertico-directory-up))))
+      (keymap-set vertico-map "M-h" 'vertico-directory-up)))
 
+  ;; Turn off the built-in fido-vertical-mode and icomplete-vertical-mode, if
```

Comment:
Do we need something similar in `crafted-defaults-config.el` since we can't guarantee the order in which the modules are loaded by the user?

Reply:
Thanks for the feedback. Good point. I now used the same logic as with the evil related-settings before. Check with `featurep`, but don't act until the respective module has been loaded. That should be the right way to deal with load order issues, right?